hazelcast/src/main/java/com/hazelcast/spi/OperationService.java

```diff
@@ -68,7 +68,10 @@
     /**
      * Invokes a set of operations on each partition.
      * <p>
-     * This method blocks until the operation completes.
+     * This method blocks until the operations complete.
+     * <p>
+     * If the operations have sync backups, this method will <b>not</b> wait for their completion.
+     * Instead, it will return once the operations are completed on the given {@code partitions}.
```

Comment:
maybe we can clarify a bit more like:
`Instead, it will return once the operations are completed on *primary replicas of* the given {@code partitions}.`